docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+---
+title: Marketplace Product Option feature integration
+last_updated: Jul 28, 2021
+Description: This document describes the process how to integrate the Marketplace Product Option feature into a Spryker project.
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Marketplace Product Option feature into a Spryker project.
+
+
+## Install feature core
+
+Follow the steps below to install the Marketplace Product Option feature core.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME | VERSION | LINK |
+| --------------- | ------- | ---------- |
+| Spryker Core         | master      | [Spryker Core Feature Integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
+| Marketplace Merchant | master      | [Marketplace Merchant Feature Integration](/docs/marketplace/dev/feature-integration-guides/{{ page.version }}/marketplace-merchant-feature-integration.html) |
+| Marketplace Product + Marketplace Product Offer | master | [Marketplace Product + Marketplace Product Offer feature integration](/docs/marketplace/dev/feature-integration-guides/{{ page.version }}/marketplace-product-marketplace-product-offer-feature-integration.html) |
```

Comment:
Connector IG's should have a link to the main IG's, but not vise versa.
The Link can be removed.